torch/csrc/jit/passes/canonicalize.cpp

```diff
@@ -5,7 +5,7 @@ namespace torch { namespace jit {
 // Canonicalize a graph, renumbering it so that all structurally equivalent
 // graphs have same numbers.
 std::shared_ptr<Graph> Canonicalize(const std::shared_ptr<Graph>& graph) {
-  auto r = std::make_shared<Graph>(graph->scope_root());
+  auto r = std::make_shared<Graph>(graph->current_scope());
```

Comment:
This seems like a logical change?

Reply:
I think so -- basically the only previous reason to copy the scope root was to allow for scopes to be copied over. This is probably unnecessary now but it ensures that the current_scope is not set differently across two graph copies, which seems reasonable.